src/axes/categoryAxis.ts

```diff
@@ -1,15 +1,7 @@
 namespace Plottable.Axes {
-  export interface DownsampleInfo {
-    domain: string[];
-    stepWidth: number;
-  }
+  export type ITickMarkPosition = "center" | "after";
```

Comment:
In Adi's comment https://github.com/palantir/plottable/pull/3067#issuecomment-207103123 the image suggests that there are lines after each series, not just between. If we do go with "between", i'll want to remove the final gridline (should we also remove the final tick?